tools/vscode/src/providers/codelens/codelens-provider.ts

```diff
@@ -33,13 +42,31 @@ function taskCommands(uri: Uri, fn: string): Command[] {
         tooltip: "Execute this evaluation task.",
         command: "inspect.runTask",
         arguments: [uri, fn],
-      }
+      },
     ];
   }
 }
 
-class InspectCodeLensProvider implements CodeLensProvider {
-  constructor() { }
+export class InspectCodeLensProvider implements CodeLensProvider {
+  private hasInspectImport(document: TextDocument): {
+    hasImport: boolean;
+    alias?: string;
+  } {
+    const text = document.getText();
+    // Handle multiline imports by removing newlines between parentheses
+    const normalizedText = text.replace(/\(\s*\n\s*([^)]+)\s*\n\s*\)/g, "($1)");
```

Comment:
Could you pull these regexes out into constants since they don't need to be recompiled each time the function is used?

Reply:
done